src/TextUI/CliArguments/Configuration.php

```diff
@@ -956,6 +962,20 @@ public function executionOrderDefects(): int
         return $this->executionOrderDefects;
     }
 
+    public function hasFailOnEmptyTestSuite(): bool
+    {
+        return $this->failOnEmptyTestSuite !== null;
+    }
+
+    public function failOnEmptyTestSuite(): bool
+    {
+        if ($this->failOnEmptyTestSuite === null) {
```

Comment:
```suggestion
        if (!$this->hasFailOnEmptyTestSuite()) {
```

I think this will improve the intent.

Reply:
You are right, but this will then look very different from the other methods in this class - that's why I stuck with the same design.